frontend/src/components/forms/EditLog.tsx

```diff
@@ -356,14 +361,15 @@ const EditLog = ({
                 </Col>
                 <Col>
                   <FormControl isRequired isInvalid={residentError} mt={4}>
-                    <FormLabel>Resident</FormLabel>
+                  <FormLabel>Residents</FormLabel>
                     <Select
                       options={residentOptions}
+                      isMulti
+                      closeMenuOnSelect={false}
                       placeholder="Select Resident"
-                      onChange={handleResidentChange}
-                      styles={selectStyle}
-                      defaultValue={residentOptions.find(
-                        (item) => item.label === logRecord.residentId,
+                      onChange={handleResidentsChange}
+                      defaultValue={residentOptions.filter(
+                        (item) => logRecord.residents.includes(item.label),
                       )}
```

Comment:
Looks like the styling on this Edit Resident dropdown (you can see its different on the form), so adding `styles={selectStyle}` should solve this